resources/metadata/gradleExecuteBuild.yaml

```diff
@@ -33,6 +33,13 @@ spec:
           - STEPS
           - STAGES
           - PARAMETERS
+      - name: publish
```

Comment:
we can add a `default : false` to make sure that publish is only called explicitly when required

Reply:
> we can add a `default : false` to make sure that publish is only called explicitly when required

default value added